x-pack/legacy/plugins/siem/cypress/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+// https://github.com/cypress-io/eslint-plugin-cypress#usage
```

Comment:
Surprised you're not getting an error as JSON doesn't have comments, this isn't technically valid JSON.

I would be careful about having non-JSON in a `.json` file.

Reply:
Good call. I'll add it to the cypress README.md file. Might be better to have something about linting in the README anyways.